packages/xsel/build.sh

```diff
@@ -0,0 +1,29 @@
+TERMUX_PKG_HOMEPAGE=http://www.kfish.org/software/xsel/
+TERMUX_PKG_DESCRIPTION="Command-line program for getting and setting the contents of the X selection"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="Doron Behar <me@doronbehar.com>"
+TERMUX_PKG_VERSION=1.2.0.20190505
+_commit=24bee9c7f4dc887eabb2783f21cbf9734d723d72
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_DEPENDS="libx11"
+TERMUX_PKG_BUILD_DEPENDS="libxt"
+
+termux_step_extract_package() {
+	mkdir -p $TERMUX_PKG_SRCDIR
+	cd $TERMUX_PKG_SRCDIR
+	git clone https://github.com/kfish/xsel
+	cd xsel
+	git checkout $_commit
+	./autogen.sh --prefix=$TERMUX_PREFIX
+}
+
+termux_step_make() {
+	cd $TERMUX_PKG_SRCDIR/xsel
+	make
+}
```

Comment:
For autotools build, make step shouldn't be overridden.

Reply:
Both 2 things caused serious problem with build where host compiler is used instead of cross:
```
make  all-am
make[1]: Entering directory '/home/builder/.termux-build/xsel/src/xsel'
gcc -DHAVE_CONFIG_H -I.     -g -O2 -fno-strict-aliasing -Wall -Werror -g -std=gnu99 -Wdeclaration-after-statement -Wno-unused -MT xsel.o -MD -MP -MF .deps/xsel.Tpo -c -o xsel.o xsel.c
mv -f .deps/xsel.Tpo .deps/xsel.Po
gcc  -g -O2 -fno-strict-aliasing -Wall -Werror -g -std=gnu99 -Wdeclaration-after-statement -Wno-unused   -o xsel xsel.o  -lX11 
make[1]: Leaving directory '/home/builder/.termux-build/xsel/src/xsel'
make[1]: Entering directory '/home/builder/.termux-build/xsel/src/xsel'
```